lib/fidor_api/model/standing_order.rb

```diff
@@ -0,0 +1,50 @@
+require 'fidor_api/model/beneficiary_helper'
+
+module FidorApi
+  module Model
+    class StandingOrder < Model::Base
+      include BeneficiaryHelper
+
+      attribute :id,           :string
+      attribute :account_id,   :string
+      attribute :external_uid, :string
+      attribute :amount,       :integer
+      attribute :currency,     :string
+      attribute :subject,      :string
+      attribute :beneficiary,  :json
+      attribute :schedule,     :json
+
+      attribute_decimal_methods :amount
+
+      def beneficiary=(value)
```

Comment:
Is it possible to move this to the `BeneficiaryHelper`?

Reply:
Directly, no. I believe it's the 'attribute :beneficiary,  :json' part that does not agree with this(the result is that define_methods_for is not executed). I tried moving the 'beneficiary' and 'routing_type' into a class to be extended by the standing_order, but then problems happened with the 'defined_methods_for' in the helper. At the end I thought that it's best not to risk breaking the transfers and left it like it's now. I can look more into it, if you like.

Reply:
Ok. Let's keep it as it is for now.
I plan to rework parts of this in near future.